src/test/java/LoginTest.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.events.EventFiringWebDriver;
+import pages.CustomerLoginPage;
+import pages.LandingPage;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class LoginTest {
+
+    private static WebDriver driver;
+
+    @Before
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver",
+                "src/main/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.get("https://magento.softwaretestingboard.com/");
+
+    }
+    @Test
+    public void  testSingIn () {
```

Comment:
What this test method is doing? Testing sign in or registration? With valid or invalid data?
The test method name should include all information what is tested inside it.
So it should be something like testSignInWithValidCredentials() or format like whatIsUnderTest_Conditions_ExpectedResults => loginForm_ValidUserCredentials_UserSuccessfullyLoggedIn()